client/src/components/Donation/donation-modal.tsx

```diff
@@ -71,6 +72,22 @@ const GetCommonDonationText = ({ ctaNumber }: { ctaNumber: number }) => {
   }
 };
 
+const RenderIlustration = ({
+  recentlyClaimedBlock
+}: {
+  recentlyClaimedBlock: RecentlyClaimedBlock;
+}) => {
+  const showModalBears = useFeature('show-modal-bears').on;
+  if (showModalBears && recentlyClaimedBlock !== null) {
+    if (recentlyClaimedBlock !== null) return <BearBlockCompletion />;
+    else return <BearProgressModal />;
```

Comment:
Is this correct? It looks like the nested `if` condition will always run, because `recentlyClaimedBlock` is not `null` in the outer if.

Reply:
Thanks for catching that.